### ISSUE TITLE
Question about multiple view fusion?

### ISSUE THREAD

**Issue by Xiaolong-RRL** [OpenGVLab/VideoMAEv2#7]
请问论文表格6(a)中的multiple view fusion是什么意思呢？

**Comment by congee524** [comment]
可以把 view 理解成视频中采样出来的一个 16x224x224 的 clip。训练时从就从视频中取 1 个 view，然后放到模型里出特征再分类。
测试的时候，大家会从同一个视频里在时间和空间维度上采样不同的 views 来 ensemble。
比如 `5x3` : 对同一视频在时序维度上取 5 个不同的 clip，每个 clip 都是 16 帧；在空间位置上也划分出 3 个 224x244 的位置（比如最左边、中间、最右边），这样就有 15 个 16x224x244 的 clip。每个 clip 放到模型里分类得到最后的 logits，把 logits softmax 之后加在一起取最大的就是预测的视频类别。

**Comment by Xiaolong-RRL** [comment]
明白了，感谢解答~
